test/test_scanner.py

```diff
@@ -814,6 +827,12 @@ def test_binaries(self, binary, package, version, are_in, not_in):
                         "gnutls-cli",
                         "2.3.11",
                     ),
+                    (
+                        "https://kojipkgs.fedoraproject.org/packages/expat/2.2.1/1.fc24/x86_64/",
```

Comment:
Could this be grouped together with the other expat tests in the file, instead of being placed under gnutls?

Reply:
Sure, Terri. 
Updated this as well as other PRs to resolve this issue.